src/encoded/regulome_search.py

```diff
@@ -63,7 +64,7 @@ def regulome_search(context, request):
         return {
             '@context': '/terms/',
             '@id': '/regulome-search',
-            'assembly': 'GRCh37',
+            'assembly': 'GRCh38',
```

Comment:
Seems like everywhere else GRCh37 turned into hg19, but not here?

Reply:
I'm not 100% confident about my python changes, but I think this is one of the two places where I'm trying to change the default to be GRCh38 instead of GRCh37/hg19. We want everything to default to GRCh38 now.